js/main.js

```diff
@@ -22,6 +22,10 @@ Site = {
       _this.Noticias.init();
     }
 
+    if ($('body').hasClass('page-home') || $('body').hasClass('page-citas')) {
+      _this.layoutMasonry();
```

Comment:
cus we compartmentalize javascript as much as possible. Site should only call other objects really. It could be a `Site.Layout` object or specific to that layout style